### PR TITLE
Add conference name in logs, output pdf and appsettings.json

### PR DESCRIPTION
**Pull request by leblemelvin**
New keywords in appsettings.json 

```
"KEYWORD_REPLACED_CONFNAME": "ConfName",
"CONFERENCE_NAME": "Entrepreneurship during COVId-19"
```

### CODE REVIEW

File: BusinessLayer/InventoryCsvManager.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public Contact AddSerialNumber(Contact contact)
             contact.SerialNumber = GetLastSerialNumber() + 1;
             using (var streamWriter = new StreamWriter(_config["INVENTORY_PATH"], true))
             {
-                var line = "\n" + contact.SerialNumber + "," + contact.Name + "," + contact.Mail + "," +
+                var line = "\n" + contact.SerialNumber + "," + contact.Name + "," + contact.Mail + "," + _config["CONFERENCE_NAME"] +
```
Comment (sewelseb) [inline BusinessLayer/InventoryCsvManager.cs]: the date of the conference is still missing

**Review by sewelseb — CHANGES_REQUESTED** [review]

File: BusinessLayer/InventoryCsvManager.cs (hunk 1 above)
Comment (leblemelvin) [reply BusinessLayer/InventoryCsvManager.cs]: The changes request are here, there is just the date which is on the following line

image

Comment (sewelseb) [reply BusinessLayer/InventoryCsvManager.cs]: no, this is the date of the emitssion of the certificate, not the date of the conference. you need two different dates

File: TonicCertificateGenerator/Program.cs [hunk 2]
```diff
@@ -25,8 +25,8 @@ private static void Main(string[] args)
                 contactManager.SetOutputDirectoryForFiles(config["OUTPUT_DIR"]);
                 var listContactFilepathPair = contactManager.GetDocumentForAllContacts();
                 var mailManager = (IMailManager) serviceProvider.GetService(typeof(IMailManager));
-                foreach (var contactFilepathPair in listContactFilepathPair)
-                    mailManager.SendEmailToContactWithAttachmnent(contactFilepathPair);
+                //foreach (var contactFilepathPair in listContactFilepathPair)
```
Comment (sewelseb) [inline TonicCertificateGenerator/Program.cs]: why comments?

Comment (leblemelvin) [reply TonicCertificateGenerator/Program.cs]: Comments is removed.
It was a small mistake.

**Pull request merged**